advanced/index.md

```diff
@@ -49,8 +49,8 @@ Explore the structure and way change is preserved in Git.
 ---
 
 {% capture lab %}
-1. Understand the way Git preserves history and data objects
-2. Utilize shorthand for navigation commit history
+1. Understand the way Git preserves data
```

Comment:
I agree. My thought was that the act of preserving is the fundamental activity, and that how one preserves history is encapsulated in the story of how one preserves data. But I'm happy to revert this change if the new sentence is too general (or otherwise worse than the original).